docs/userGuide/syntax/attributes.mbdf

```diff
@@ -0,0 +1,65 @@
+## Classes, Attributes & Identifiers
+
+Most markdown syntax above this section supports adding classes, attributes and identifiers
+using [pandoc](https://pandoc.org/MANUAL.html) syntax without the need for a wrapper html element.
+
+The syntax is `{.class-name attribute="value" attribute=value #id}`, which is placed at different locations depending
+on the type of markdown.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+Apply classes, attributes, identifiers to block level markdown (eg. paragraphs, headings)
+by leaving a space before '{' {.text-success #attribute-example}
+
+#### heading {.text-info}
+
+--- {.border-danger}
+
+Apply the same to inline markdown (eg. bold text) by
+omitting the **space**{.text-primary .bg-light attribute=value}
+</variable>
+</include>
```

Comment:
Side question: what if I want to apply a style to multiple words inside a paragraph that doesn't have any markdown symbol around them? No shorthand syntax for that I guess.

Reply:
yep don't think there's any shorthand for that unfortunately